decentraland/components/presentational/__snapshots__/BuyLand.test.js.snap

```diff
@@ -41,3 +41,261 @@ exports[`BuyLand renders the component 1`] = `
   </Component>
 </Component>
 `;
+
+exports[`BuyLand renders the component initial state - before start account setup 1`] = `
+<Component
+  style={
+    Object {
+      "alignItems": "center",
+      "backgroundColor": "white",
+      "flex": 1,
+      "justifyContent": "center",
+    }
+  }
+>
+  <LandForSale
+    landForSale={
+      Object {
+        "asset": Object {
+          "id": "0123456789",
+          "img": "https://api.decentraland.org/v1/parcels/0/0/map.png",
+          "name": "Sample Parcel",
+          "type": "PARCEL",
+        },
+        "id": "2",
+        "priceMana": 0,
+        "priceUSD": 0,
+      }
+    }
+  />
+  <Component
+    style={
+      Object {
+        "flexDirection": "row",
+        "marginTop": 66.7,
+      }
+    }
+  >
+    <Button
+      onPress={[Function]}
+      title="Buy"
+    />
+  </Component>
+</Component>
+`;
+
+exports[`BuyLand renders the component waiting for setup - waiting for ETH funding 1`] = `
+<Component
+  style={
+    Object {
+      "alignItems": "center",
+      "backgroundColor": "white",
+      "flex": 1,
+      "justifyContent": "center",
+    }
+  }
+>
+  <LandForSale
+    landForSale={
+      Object {
+        "asset": Object {
+          "id": "0123456789",
+          "img": "https://api.decentraland.org/v1/parcels/0/0/map.png",
+          "name": "Sample Parcel",
+          "type": "PARCEL",
+        },
+        "id": "2",
+        "priceMana": 0,
+        "priceUSD": 0,
+      }
+    }
+  />
+  <Component
+    style={
+      Object {
+        "flexDirection": "row",
+        "marginTop": 66.7,
+      }
+    }
+  >
+    <Button
+      disabled={true}
+      onPress={[Function]}
+      title="Buy"
+    />
+  </Component>
+  <Component
+    style={
+      Object {
+        "flexDirection": "row",
+      }
+    }
+  >
+    <Component>
+      Generating account...
+    </Component>
+  </Component>
+</Component>
+`;
+
+exports[`BuyLand renders the component waiting for setup - waiting for MANA funding 1`] = `
+<Component
+  style={
+    Object {
+      "alignItems": "center",
+      "backgroundColor": "white",
+      "flex": 1,
+      "justifyContent": "center",
+    }
+  }
+>
+  <LandForSale
+    landForSale={
+      Object {
+        "asset": Object {
+          "id": "0123456789",
+          "img": "https://api.decentraland.org/v1/parcels/0/0/map.png",
+          "name": "Sample Parcel",
+          "type": "PARCEL",
+        },
+        "id": "2",
+        "priceMana": 0,
+        "priceUSD": 0,
+      }
+    }
+  />
+  <Component
+    style={
+      Object {
+        "flexDirection": "row",
+        "marginTop": 66.7,
+      }
+    }
+  >
+    <Button
+      disabled={true}
+      onPress={[Function]}
+      title="Buy"
+    />
+  </Component>
+  <Component
+    style={
+      Object {
+        "flexDirection": "row",
+      }
+    }
+  >
+    <Component>
+      Generating account...
```

Comment:
Don't we want this to be something dynamic, like showing "Waiting for MANA funding" or whatever the toast says?

Reply:
The test case was buggy. Fixed.

Reply:
Thanks! Resolving this

Reply:
Are you sure it was the test case that was buggy? Because I think this is also what I saw as a user

Reply:
Both were. The test suite wasn't looking to redux `account` prop (that is used to know if the account was generated [!= null]). And the app too because the `setAccount` was being called at the end of the setup process. The output of both (test and app) were the same.

Reply:
Ah right, got it. Thanks for the explanation!